package/post-install.sh

```diff
@@ -168,10 +168,13 @@ PHP_MAJOR_MINOR=$(invoke_php -r 'echo PHP_MAJOR_VERSION;').$(invoke_php -r 'echo
 PHP_CFG_DIR=$(invoke_php -i | grep 'Scan this dir for additional .ini files =>' | sed -e 's/Scan this dir for additional .ini files =>//g' | head -n 1 | awk '{print $1}')
 
 PHP_THREAD_SAFETY=$(invoke_php -i | grep 'Thread Safety' | awk '{print $NF}' | grep -i enabled)
+PHP_DEBUG_BUILD=$(invoke_php -i | grep 'Debug Build => ' | awk '{print $NF}' | grep -i yes)
 
 VERSION_SUFFIX=""
 if [[ -n $PHP_THREAD_SAFETY ]]; then
     VERSION_SUFFIX="-zts"
+elif [[ -n $PHP_DEBUG_BUILD ]]; then
```

Comment:
I think this is correct but I want to double check with you to make sure I am understanding it correctly.

`$PHP_DEBUG_BUILD` means php's debug build (like in `--enable-debug`), while for our extension `-debug` means pretty much `CFLAGS = -g`.

So here we are saying that when we have debug builds, no matter what, we want also our symbols (which is logical). While this would not impact the actual "production" and the respective verification.

Am I correct?

Reply:
Thanks for double checking this!

> $PHP_DEBUG_BUILD means php's debug build (like in --enable-debug)

Correct.

> while for our extension -debug means pretty much CFLAGS = -g.

All three of the `.so` build flavors we ship ([nts](https://github.com/DataDog/dd-trace-php/blob/f0589aad7996a10397593a3592cd714a99982975/.circleci/config.yml#L866), [zts](https://github.com/DataDog/dd-trace-php/blob/f0589aad7996a10397593a3592cd714a99982975/.circleci/config.yml#L878), [debug](https://github.com/DataDog/dd-trace-php/blob/f0589aad7996a10397593a3592cd714a99982975/.circleci/config.yml#L872)) include debug symbols. 

When a PHP flavor is built with `--enable-debug` and/or `--enable-zts`, it will change the build ID. Our extension must be built against the proper PHP flavor, otherwise we'll run into errors like the one identified in #906.  The `switch-php` script also updates the symlink to `php-config` which will ensure that the `.so` will build with the proper build ID.

A while ago we had a bug where our debug builds only included debug symbols but didn't have the correct build ID for PHP flavors built with `--enable-debug` but now they build with the proper build ID. 

The change in this PR fixes another bug where the post-install script did not install the the proper `.so` for the debug flavor of PHP. Now it properly matches the build ID for debug builds of PHP.